MusicService/PlayCommand.cs

```diff
@@ -68,11 +68,69 @@ public async Task PlayCommandAsync([Summary("vibe", "The name or youtube link of
             embed.WithAuthor($"✅ Vibe Set by {Context.User.Username}");
 
         embed.WithTitle(setVibe.Title)
-             .WithThumbnailUrl(Context.User.GetAvatarUrl())
+             .WithThumbnailUrl(GetYoutubeThumbnailUrl(setVibe.Uri))
              .AddField("Channel", setVibe.Author, true)
              .AddField("Duration", setVibe.Duration.ToString("d':'hh':'mm':'ss"), true)
              .AddField("Position", pos, true);
 
         await RespondAsync(embed: embed.Build());
     }
+
+    private static string GetYoutubeThumbnailUrl(Uri? videoUrl)
+    {
+        string fallbackThumbnailUrl = "";
```

Comment:
```suggestion
        var fallbackThumbnailUrl = "";
```
Use var instead of the types.